ONSdigital/eq-author-app
# Ear 572 skip logic prep

### What is the context of this PR?

Created a new Logic page according to New design
Got the new Logic menu working with Routing and ready to accommodate skip logic.
(currently defaults to Routing and Skip menu item is hidden)

I've Moved the routing folder down into the new Logic folder
Created a new Logic page where Routing used to be and inserted the new design sidebar menu ready to accept new skip logic code.

https://collaborate2.ons.gov.uk/jira/browse/EAR-572

The new logic folder will accommodate the skip folder and code when ready

### How to review 

- Check the routing tab in the header now says 'Logic'
- Click on it and it should default to routing
- Currently the new sideBar menu will only show 'Routing Logic'
- The menu has commented out code for working 'Skip Logic' menu item, so please accept that there is some commented code in this PR.
- Make sure Routing is still working as expected

### What to do after everything is green
